mgmt/src/models/internal/natconfig/table_extend.rs

```diff
@@ -705,44 +704,44 @@ mod tests {
         }
     }
 
-    fn prefix_oracle(addr: &IpNet, prefixes: &IpRTrieSet, excludes: &IpRTrieSet) -> bool {
-        !excludes.contains(addr) && prefixes.contains(addr)
-    }
-
-    #[test]
-    fn test_bolero_collapse_prefix_lists() {
-        let generator = PrefixExcludeAddrsGenerator {
-            prefix_max: 100,
-            exclude_max: 100,
-            addr_count: 1000,
-        };
-        bolero::check!()
-            .with_generator(generator)
-            .for_each(|data: &PrefixExcludeAddrs| {
-                let PrefixExcludeAddrs {
-                    prefixes,
-                    excludes,
-                    addrs,
-                } = data;
-                let mut prefixes_trie = IpRTrieSet::new();
-                let mut excludes_trie = IpRTrieSet::new();
-                let mut collapsed_prefixes_trie = IpRTrieSet::new();
-                for prefix in prefixes {
-                    prefixes_trie.insert(IpNet::from(*prefix));
-                }
-                for exclude in excludes {
-                    excludes_trie.insert(IpNet::from(*exclude));
-                }
-                let collapsed_prefixes = collapse_prefix_lists(prefixes, excludes).unwrap();
-                for prefix in collapsed_prefixes {
-                    collapsed_prefixes_trie.insert(IpNet::from(prefix));
-                }
-                for addr in addrs {
-                    let addr_net = IpNet::from(*addr);
-                    let oracle_result = prefix_oracle(&addr_net, &prefixes_trie, &excludes_trie);
-                    let collapsed_result = collapsed_prefixes_trie.contains(&addr_net);
-                    assert_eq!(oracle_result, collapsed_result);
-                }
-            });
-    }
+    // fn prefix_oracle(addr: &IpNet, prefixes: &IpRTrieSet, excludes: &IpRTrieSet) -> bool {
```

Comment:
Why is this test still commented out?

Reply:
Good catch!. Forgot to ask too.

Reply:
@mvachhar What do you mean? You were the one to request and write it (c3ad0cef4711390d8ce6a951386b90db0352207b) 😅

Reply:
I commented it out because it was the last test I needed to convert over to the new prefix map and I wanted to run this through CI before I did the last step

Reply:
I can't even remember what is was supposed to do :(

Reply:
Ahh, right, this tests the prefix calculator, we need this test.